tutorials/Bayesian_Logistic_Regression.ipynb

```diff
@@ -9,8 +9,6 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    import matplotlib.pyplot as plt
</pre><p>Do you want to add the colab related instruction in this diff itself to minimize iterations?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>